lib/sidekiq/cli.rb

```diff
@@ -62,6 +62,11 @@ def run(boot_app: true)
       ver = Sidekiq.redis_info["redis_version"]
       raise "You are connecting to Redis v#{ver}, Sidekiq requires Redis v4.0.0 or greater" if ver < "4"
 
+      maxmemory_policy = Sidekiq.redis_info["maxmemory_policy"]
+      if maxmemory_policy != "noeviction"
```

Comment:
There will be a lot of these in development.  I'd suggest you only print this if env != "development".

Reply:
Taking a look at `redis.conf` in various versions, I believe `noeviction` is the default:

- [4.0](https://github.com/redis/redis/blob/ff6db5f176caa426135bf9e603a1eb6f6dea4f83/redis.conf#L588-L590)
- [5.0](https://github.com/redis/redis/blob/a767d84a7273ea05bbfbb23f4eef63af7f4a613d/redis.conf#L595-L597)
- [6.0](https://github.com/redis/redis/blob/25214bd7dc2f4c995d76020e95180eb4e6d51672/redis.conf#L889-L891)

But appreciate that it's a somewhat meaningless check in development. I'll add the additional condition 👍

Reply:
Good catch, if it’s the default in the brew formula then I’m ok with it as is.

Reply:
It appears that the Homebrew formulae don't modify `maxmemory_policy`.

Here's `4.0`:
https://github.com/Homebrew/homebrew-core/blob/fdb83fcfb482e5ed1f1c3c442a85b99223fcabeb/Formula/redis%404.0.rb#L27-L31

Here's `6.0`:
https://github.com/Homebrew/homebrew-core/blob/fdb83fcfb482e5ed1f1c3c442a85b99223fcabeb/Formula/redis.rb#L30-L34